audit_history/tests/test_audit_history.py

```diff
@@ -16,25 +16,25 @@ def test_empty_history_after_save(self):
 
     def test_history_after_save_with_audit(self):
         self.blog_post.save_with_audit_record(None, self.history_event, payload=self.payload)
-        self.assertEqual(self.payload[1], self.blog_post.history[0].get('payload').get(1))
+        self.assertEqual(self.payload['a'], self.blog_post.history[0]['payload']['a'])
 
     def test_save_with_audit_record_with_staff_user(self):
         self.blog_post.save_with_audit_record(self.user, self.history_event, payload=self.payload)
         self.assertEqual(self.user_name, self.blog_post.history[0]['actor']['name'])
         self.assertEqual(self.user_email, self.blog_post.history[0]['actor']['email'])
         self.assertEqual(True, self.blog_post.history[0]['actor']['is_staff'])
-        self.assertEqual(self.payload[1], self.blog_post.history[0]['payload'][1])
-        self.assertEqual(self.payload[2], self.blog_post.history[0]['payload'][2])
+        self.assertEqual(self.payload['a'], self.blog_post.history[0]['payload']['a'])
+        self.assertEqual(self.payload['b'], self.blog_post.history[0]['payload']['b'])
 
     def test_save_with_audit_record_without_user(self):
         self.blog_post.save_with_audit_record(None, self.history_event, payload=self.payload)
         self.assertEqual(None, self.blog_post.history[0]['actor'])
-        self.assertEqual(self.payload[1], self.blog_post.history[0]['payload'][1])
-        self.assertEqual(self.payload[2], self.blog_post.history[0]['payload'][2])
+        self.assertEqual(self.payload['a'], self.blog_post.history[0]['payload']['a'])
+        self.assertEqual(self.payload['b'], self.blog_post.history[0]['payload']['b'])
 
     def test_n_usages_of_save_with_audit(self):
         count = randint(1, 100)
-        for _ in xrange(count):
+        for _ in range(count):
```

Comment:
Just out of curiosity: why this change?

Reply:
Because in python3 range == xrange in python2
https://github.com/espressopp/espressopp/issues/111

Reply:
Ah yes, makes sense! Forgot about that.